src/lib/Scenes/Show/Components/Artists/__tests__/Artists-tests.tsx

```diff
@@ -28,8 +28,10 @@ const renderTree = () =>
         }
       `}
       mockResolvers={{
-        Show: () => ShowFixture,
-        Artist: () => ({ name: "Hans Hofmann", is_followed: false, id: "hans-hofmann" }),
+        Show: () => ({
+          ...ShowFixture,
+          artists: () => ShowFixture.show.artists,
```

Comment:
TIL about [`MockList`](https://www.apollographql.com/docs/graphql-tools/mocking.html#MockList) while trying to figure out the right way to do this.